apps/demos/Demos/Splitter/Overview/React/App.tsx

```diff
@@ -1,15 +1,22 @@
-import React from 'react';
+import React, { useEffect } from 'react';
```

Comment:
useEffect is unused here. Could you please also check eslint fixable errors after your changes?